tests/chart_tests/test_sts_volume_claim_annotations.py

```diff
@@ -0,0 +1,138 @@
+import pytest
+
+from tests import supported_k8s_versions
+from tests.chart_tests.helm_template_generator import render_chart
+
+
+@pytest.mark.parametrize(
+    "kube_version",
+    supported_k8s_versions,
+)
+class TestStatefulSetsAnnotations:
+    def test_sts_with_no_annotations(self, kube_version):
+        """Test all sts for the volume claim templates annotations"""
+        docs = render_chart(
+            kube_version=kube_version,
+            values={},
+            show_only=[
+                "charts/elasticsearch/templates/master/es-master-statefulset.yaml",
+                "charts/elasticsearch/templates/data/es-data-statefulset.yaml",
+                "charts/alertmanager/templates/alertmanager-statefulset.yaml",
+                "charts/astronomer/templates/registry/registry-statefulset.yaml",
+                "charts/prometheus/templates/prometheus-statefulset.yaml",
+                "charts/stan/templates/statefulset.yaml",
+            ],
+        )
+
+        assert len(docs) == 6
+
+        with pytest.raises(Exception) as exec_info:
+            for doc in docs:
+                if doc["spec"]["volumeClaimTemplates"][0]["metadata"]["annotations"]:
+                    raise Exception("Empty Annotations")
+            assert exec_info.value.args[0] == "Empty Annotations"
+            assert str(exec_info.value) == "Empty Annotations"
```

Comment:
Do we need this `pytest.raises` context manager? Can we `assert not doc["spec"]["volumeClaimTemplates"][0]["metadata"]["annotations"]` or something with simpler syntax?